examples/sagemaker-tensorflow/sagemaker-merlin-tensorflow.ipynb

```diff
@@ -0,0 +1,1392 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    workflow = Workflow.load("/tmp/ensemble/0_transformworkflow/1/workflow/")
</pre><p>Getting the following error?</p><p> </p><p> </p><pre class="ql-syntax" spellcheck="false">---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
Input In [33], in &lt;cell line: 1&gt;()
----&gt; 1 Workflow.load("/tmp/ensemble/0_transformworkflow/1/workflow/")

File /usr/local/lib/python3.8/dist-packages/nvtabular/workflow/workflow.py:392, in Workflow.load(cls, path, client)
    389     warnings.warn(f"Loading workflow generated on {expected}")
    391 # load up the workflow object di
--&gt; 392 workflow = cloudpickle.load(fs.open(fs.sep.join([path, "workflow.pkl"]), "rb"))
    393 workflow.client = client
    395 # we might have been copied since saving, update all the stat ops
    396 # with the new path to their storage locations

File /usr/lib/python3.8/enum.py:339, in EnumMeta.__call__(cls, value, names, module, qualname, type, start)
    314 """
    315 Either returns an existing member, or creates a new enum class.
    316 
   (...)
    336 <code>type</code>, if set, will be mixed in as the first base class.
    337 """
    338 if names is None:  # simple value lookup
--&gt; 339     return cls.__new__(cls, value)
    340 # otherwise, functional API: we're creating a new Enum type
    341 return cls._create_(
    342         value,
    343         names,
   (...)
    347         start=start,
    348         )

File /usr/lib/python3.8/enum.py:663, in Enum.__new__(cls, value)
    661 ve_exc = ValueError("%r is not a valid %s" % (value, cls.__name__))
    662 if result is None and exc is None:
--&gt; 663     raise ve_exc
    664 elif exc is None:
    665     exc = TypeError(
    666             'error in %s._missing_: returned %r instead of None or a valid member'
    667             % (cls.__name__, result)
    668             )

ValueError: 'id' is not a valid Tags

</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Merlin/pull/692/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
@angmc Which container did you use? I just realized I used `22.09` and this hasn't been tested with `22.10`. I'll test it with `22.10`.

Reply:
<p>Yeah not sure what the problem was but it did run alright with the correct container. </p>